src/rule-transform/rule-content/get-rule-body.ts

```diff
@@ -34,3 +58,54 @@ function stripReferences({ body, markdownAST }: Args): string {
   }
   return body.substr(0, offset);
 }
+
+function addInputAspects(
+  frontmatter: AtomicRuleFrontmatter,
+  body: string,
+  supportKeys: AccessibilitySupport
+): string {
+  frontmatter.input_aspects.forEach((aspect) => {
+    if (aspect.toLowerCase() in supportKeys) {
+      body = updateAccessibilitySupportInBody(
+        body,
+        supportKeys[aspect.toLowerCase()]
+      );
+    }
+  });
+  return body;
+}
+
+function updateAccessibilitySupportInBody(
+  body: string,
+  support: string
+): string {
+  const supportIndex = body.search(/\n###\s+Accessibility\s+Support\b/m);
```

Comment:
Is it enforced somewhere that if a rule has accessibility support it will be exactly named "Accessibility Support"? Just want to make sure this can't be easily renamed and break this automation, or someone making a typo, e.g. "Acessibility Support"

I think anyway "Accessibility Support" and all other hardcoded strings, e.g. "There are no accessibility support issues known." should be constants, something along the lines

```
const SECTIONS = {
  ACCESSIBILITY_SUPPORT: {
    HEADING_PATTERN: /\n###\s+Accessibility\s+Support\b/mi,
    HEADING: "Accessibility Support"
    NO_ISSUES_TEXT: "There are no accessibility support issues known."
  }
};
```